### PR TITLE
Add feature and update site

### PR DESCRIPTION
**Pull request by GloriaWa**
add feature and update site

### CODE REVIEW

File: checker-framework-eclipse-update-site/manual/index.html [hunk 1]
```diff
@@ -0,0 +1,338 @@
+<!DOCTYPE html>
```
Comment (wmdietl) [inline checker-framework-eclipse-update-site/manual/index.html]: Is this manual for a previous version of the Eclipse plug-in?
Did you adapt the instructions to the new plug-in behavior?
You should also attribute sources, e.g. in a README file.

File: checker-framework-eclipse-update-site/manual/index.html [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+<html>
```
Comment (wmdietl) [inline checker-framework-eclipse-update-site/manual/index.html]: Can you remove the binary `.jar` files from the commit?
Aren't these files rebuilt automatically?

File: checker-framework-eclipse-update-site/manual/index.html [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+<head>
```
Comment (wmdietl) [inline checker-framework-eclipse-update-site/manual/index.html]: I would not commit this html file and the images. Instead, we should work on an up-to-date manual in a more suitable format, e.g. markdown.

**Review by wmdietl — CHANGES_REQUESTED** [review]

File: checker-framework-eclipse-update-site/manual/index.html (hunk 2 above)
Comment (GloriaWa) [reply checker-framework-eclipse-update-site/manual/index.html]: Removed. Should I also remove the log.zip file?

File: checker-framework-eclipse-update-site/manual/index.html (hunk 3 above)
Comment (GloriaWa) [reply checker-framework-eclipse-update-site/manual/index.html]: Removed the manual folder.

File: checker-framework-eclipse-update-site/manual/index.html (hunk 1 above)
Comment (GloriaWa) [reply checker-framework-eclipse-update-site/manual/index.html]: Removed the manual folder.

File: .classpath [hunk 4]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (GloriaWa) [inline .classpath:1]: The changes may need to be discarded or changed base on pr: https://github.com/eisopux/checker-framework-eclipse/pull/8

File: .settings/org.eclipse.jdt.core.prefs [hunk 5]
```diff
@@ -1,9 +1,15 @@
 eclipse.preferences.version=1
-org.eclipse.jdt.core.compiler.codegen.targetPlatform=12
```
Comment (GloriaWa) [inline .settings/org.eclipse.jdt.core.prefs:2]: The changes may need to be discarded or changed base on pr: #8

File: checker-framework-eclipse-update-site/manual/index.html (hunk 2 above)
Comment (wmdietl) [reply checker-framework-eclipse-update-site/manual/index.html]: Sure, committing a log doesn't seem useful.

**Comment by wmdietl** [comment]
For the update site, instead of adding the `.jar` files, please add a section in the README file about how to build the necessary files for the update site.

**Review by wmdietl — CHANGES_REQUESTED** [review]
Please undo the changes to the top-level files (they moved)
Please remove the .jar files.
The plugin is named "lsp_checkerframework.project". Should the feature use something along those lines instead of "checker.framework.eclipse.feature"?

**Comment by wmdietl** [comment]
> Please undo the changes to the top-level files (they moved)
> Please remove the .jar files.
> The plugin is named "lsp_checkerframework.project". Should the feature use something along those lines instead of "checker.framework.eclipse.feature"?

@GloriaWa Can you please address these comments and resolve the merge issues?

**Pull request closed without merging**